torch2trt/torch2trt.py

```diff
@@ -555,10 +555,11 @@ def torch2trt(module,
                 outputs = (outputs,)
             ctx.mark_outputs(outputs, output_names)
 
-    builder.max_workspace_size = max_workspace_size
-    builder.fp16_mode = fp16_mode
+    config = builder.create_builder_config()
+    config.max_workspace_size = max_workspace_size
+    config.flags = fp16_mode << int(trt.BuilderFlag.FP16)
+    config.flags = strict_type_constraints << int(trt.BuilderFlag.STRICT_TYPES)
```

Comment:
There's also a `config.set_flag(...)` call that you can make here, which might be safer.
In that same vein, since we're using `=` instead of `|`, would this not overwrite the `config.flags` every time you're setting it (eg. `fp16_mode` would be overwritten by `strict_type_constraints`)?

Reply:
Also, might be worth taking a look at the latest master, these changes might already be handled.